lua/telescope/utils.lua

```diff
@@ -82,6 +82,59 @@ utils.quickfix_items_to_entries = function(locations)
   return results
 end
 
+utils.filter_symbols = function(opts, results)
+  if opts.symbols == nil then
+    return results
+  end
+  local valid_symbols = vim.tbl_map(string.lower, vim.lsp.protocol.SymbolKind)
+
+  local filtered_symbols = {}
+  if type(opts.symbols) == "string" then
+    opts.symbols = string.lower(opts.symbols)
+    if vim.tbl_contains(valid_symbols, opts.symbols) then
```

Comment:
Just rambling here: Does it make sense to give a small error message if you miss typed a symbol? For both string and table. I just had a typo and that resulted into no results -> no picker (which is weird) does this make sense?